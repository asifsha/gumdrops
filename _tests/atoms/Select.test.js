/* globals mount */
import React from 'react';
import Select from '../../components/atoms/Select';

const defaultProps = {
    className: 'modal-class',
    context: 'success',
    size: 'lg',
    text: '100',
    options: [
        {
            value: 'foo',
            name: 'Foot'
        },
        {
            value: 'woo',
            name: 'Woot'
        }
    ]
};

describe('Expect <Select>', () => {
    it('to render', () => {
        const wrapper = mount(<Select {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
    });
});
