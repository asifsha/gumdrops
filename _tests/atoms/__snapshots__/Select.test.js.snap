// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <Select> to render 1`] = `
<Select
  className="modal-class"
  context="success"
  customName="name"
  customValue="value"
  options={
    Array [
      Object {
        "name": "Foot",
        "value": "foo",
      },
      Object {
        "name": "Woot",
        "value": "woo",
      },
    ]
  }
  size="lg"
  text="100"
>
  <select
    className="gds-form-group__select-input modal-class gds-form-group__select-input--success gds-form-group__select-input--lg"
    text="100"
  >
    <option
      key="foo"
      value="foo"
    >
      Foot
    </option>
    <option
      key="woo"
      value="woo"
    >
      Woot
    </option>
  </select>
</Select>
`;
